installCatalog.sh

```diff
@@ -22,7 +22,7 @@ DB_URL="$3"
 DB_NAME="${4}cloudanttrigger"
 APIHOST="$5"
 WORKERS="$6"
-ACTION_RUNTIME_VERSION=${ACTION_RUNTIME_VERSION:="nodejs:12"}
+ACTION_RUNTIME_VERSION=${ACTION_RUNTIME_VERSION:="nodejs:14"}
```

Comment:
This will probably not work, as a "nodejs:14" runtime will not be available. It could be either "nodejs:12" or "nodejs:16".

Reply:
Just checked, the value is overwritten at install time. Therefore this default is fine 👍 .